apis/v1/httproute_types.go

```diff
@@ -757,6 +757,8 @@ type HTTPRouteFilter struct {
 	// Support: Extended
 	//
 	// +optional
+	//
+	// <gateway:experimental:validation:XValidation:message="Only one of percent or fraction may be specified in HTTPRequestMirrorFilter",rule="!(has(self.percent) && has(self.fraction))">
```

Comment:
A downside of having this validation at this level is that it needs to be copied to GRPCRoute: https://github.com/kubernetes-sigs/gateway-api/blob/f778afa55e254fffa255487ac16fee58955c1961/apis/v1/grpcroute_types.go#L469